lib/internal/streams/destroy.js

```diff
@@ -6,12 +6,15 @@ function destroy(err, cb) {
     this._readableState.destroyed;
   const writableDestroyed = this._writableState &&
     this._writableState.destroyed;
+  const readableErrored = this._readableState &&
```

Comment:
Nit: can we create these inside the `if` block just below? It doesn't look like they're used outside of it.